akka-distributed-data/src/main/protobuf/ReplicatedDataMessages.proto

```diff
@@ -72,7 +72,30 @@ message ORMap {
   }
   
   required ORSet keys = 1;
-  repeated Entry entries = 2;  
+  repeated Entry entries = 2;
+  repeated Entry removedEntries = 3;
+}
+
+message ORMapDeltaGroup {
+  message Entry {
+    required ORMapDeltaOp operation = 1;
+    required ORSet underlying = 2;
+    required UniqueAddress node = 3;
+    required sint32 zeroTag = 4;
```

Comment:
To pass concrete type of the `*Map` - required to properly type the `zero` used for initial value by the delta `Replicator`